src/main/java/com/agilethought/internship/sso/controller/ControllerAplication.java

```diff
@@ -48,4 +51,11 @@ public UserId createUser(@RequestBody User user) {
 		UserId userId = serviceApplication.createUser(user);
 		return userId;
 	}
+	
+	@GetMapping(value="/at-sso-api/api/v1/user", produces = "application/json")
+	@ResponseStatus(value = HttpStatus.OK)
+	public java.util.List<User> getAllUsers(){
```

Comment:
Please add "java.util.List" like a import, the method should be like: "public List<User> getAllUsers(){"